### PR TITLE
Very basic overview of Authz

### LINKED ISSUE #24: Project request: Authz

### PR DESCRIPTION
**Pull request by bvhme**
For now, this is all that I could think of about Authz for now, for us to expand upon :)

Closes #24.

### CODE REVIEW

**Comment by bvhme** [comment]
I'm also considering adding this one, however, that might be overkill

oauth2 gopher 0 5x

Cute though :) And I like designing these things.

File: projects/authz.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+abstract: OAuth 2.0 authorization service written in Go
+---
+
+# Authz
+
+An implementation of the [OAuth 2.0 implicit authorization flow (RFC6749)](https://tools.ietf.org/html/rfc6749#section-1.3.2) as a HTTP service written in Go.
```
Comment (evertlammerts) [inline projects/authz.md]: `An [OAuth 2.0](https://tools.ietf.org/html/rfc6749) compatible authorization service written in Go.`

File: projects/authz.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+This service:
+* creates signed JSON Web Token (JWS) access tokens to be used what RFC6749 calls resource servers
+* Can use in-memory state storage or be connected to external storage such as Redis
+
```
Comment (evertlammerts) [inline projects/authz.md]:
```
This service:

* Implements the implicit flow as described in [RFC6749 section 4.2](https://tools.ietf.org/html/rfc6749#section-4.2)
* Uses JSON Web Keys ([RFC7517](https://tools.ietf.org/html/rfc7517)) for key management
* Creates JSON Web Tokens ([RFC7519](https://tools.ietf.org/html/rfc7519)) using HMAC or ECDSA (HS256, HS384, HS512, ES256, ES384, ES512)
* Provides interfaces for identity providers, state storage (with implementations for single node in-memory storage and Redis), authorization providers (to map users to scopes) and client registries.
```

**Review by evertlammerts — COMMENTED** [review]
Goed werk Boris! Hier een paar suggesties voor veranderingen.

**Comment by evertlammerts** [comment]
De gopher vind ik wel leuk :-) Als je hem wilt publiceren moet er nog wel een 2.0 achter de OAuth.

**Comment by bvhme** [comment]
Like this, without the text and a bit more bespoke?

oauth2 gopher 0 25x

**Comment by evertlammerts** [comment]
I liked the first one better!

**Comment by bvhme** [comment]
@evertlammerts I've added your changes, thanks very much.

And the OAuth logo is the same for 1 and 2, so I've added the most abstract official version there is :) I hope the Gopher is happy with its 👕 shirt

**Comment by evertlammerts** [comment]
Thanks!

**Pull request merged**